src/components/Button.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import classnames from "classnames";
+
+interface IProps {
+  isInverted?: boolean;
+  text: string;
+  isColorWhite?: boolean;
+  isSmall?: boolean;
+}
+
+export const Button: FC<IProps> = ({
+  isInverted = false,
+  text,
+  isColorWhite = false,
+  isSmall = false
```

Comment:
we don't need this, this should be dynamic

Reply:
done